### PR TITLE
packer: update atom box for 2.5 release

### PR DESCRIPTION
**Pull request by scollazo**
Use 18.04 as base image

### CODE REVIEW

File: packer/templates/vagrant-box-atom/provisioning/host_vars/all [hunk 1]
```diff
@@ -24,6 +24,9 @@ atom_mysql_user_name: "atom-user"
 atom_mysql_user_pass: "ATOMPASSWORD"
 atom_mysql_user_priv: "atom.*:ALL,GRANT"
 atom_mysql_user_host: "%"
+atom_user: "vagrant"
+atom_group: "vagrant"
+atom_description: "Access to Memory vagrat box"
```
Comment (djjuhasz) [inline packer/templates/vagrant-box-atom/provisioning/host_vars/all:29]: Typo: "Access to Memory vagrant box"

File: packer/templates/vagrant-box-atom/requirements.yml [hunk 2]
```diff
@@ -26,6 +26,6 @@
   path: "roles/"
 
 - src: "https://github.com/artefactual-labs/ansible-atom"
-  version: "dev/bionic-support"
+  version: "wip/centos-php72"
```
Comment (djjuhasz) [inline packer/templates/vagrant-box-atom/requirements.yml]: I'm not sure if this is still the right branch to use?

**Review by djjuhasz — APPROVED** [review]
Looks good to me @scollazo  👍 Though note that @mamedin upgraded the Ubuntu 18.04 box to [18.04.03](https://github.com/artefactual-labs/am-packbuild/commit/44cd3092283c2337f5bab263c021e42a11080238#diff-d933baa32e5bf2ecddc2eff5747a6588) which may conflict with this

**Comment by jraddaoui** [comment]
Included in https://github.com/artefactual-labs/am-packbuild/pull/250.

**Pull request closed without merging**